ql/src/java/org/apache/hadoop/hive/ql/parse/repl/load/UpdatedMetaDataTracker.java

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
true for insert handler you could no op. Context has the precursor task but i think for the above work it will need precursor object type which will direct the change required. I think functionally the current code will work fine, so i will bank on your decision.